File: Easy-problems/rightmost-char.java
import java.io.*;
import java.util.*;


public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;

        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here

            String[] input = line.split(",");
            System.out.println(input[0].lastIndexOf(input[1]));
        }
        
        buffer.close();
    }
    
}

File: Easy-problems/rightmost-char.cs
using System;
using System.IO;

namespace CodeEval
{
    class Program
    {
        static void Main(string[] args)
        {
            using (StreamReader reader = File.OpenText(args[0]))
            {
                while (!reader.EndOfStream)
                {

                    string line = reader.ReadLine();
                    if (String.IsNullOrEmpty(line))
                        continue;
                    // do something with line

                    string[] input = line.Split(',');
                    Console.WriteLine(input[0].LastIndexOf(input[1]));
                }
            }
            //Console.ReadLine();
        }
    }
}
